misc/config_tools/schema/types.xsd

```diff
@@ -125,7 +125,7 @@ Read more about the available scheduling options in :ref:`cpu_sharing`.</xs:docu
 
 <xs:simpleType name="SerialConsoleType">
   <xs:restriction base="xs:string">
-    <xs:pattern value=".*ttyS[0-3]" />
+    <xs:pattern value=".*ttyS[\d]+" />
```

Comment:
Do you need to update the ``SERIAL_CONSOLE`` documentation in ``config.xsd``.  Here's what's currently there (note the references to ttys0-3, and they use a lowercase "s"):

```
    <xs:element name="SERIAL_CONSOLE" type="SerialConsoleOptions" default="/dev/ttyS0">
      <xs:annotation>
        <xs:documentation>Specify the host serial device used for hypervisor debugging.
This option is only valid if :option:`hv.DEBUG_OPTIONS.RELEASE` is set to ``n``.
This option impacts the content of ``vm.(legacy_vuart id="0").base`` when :option:`vm.vm_type` is ``SOS_VM``,
which specifies the PIO base for Service VM legacy vUART 0 (used for the console).
The PIO base for the Service VM's legacy vUART 0 is determined using these rules:

- If :option:`hv.DEBUG_OPTIONS.SERIAL_CONSOLE` is ``ttys0``, ``ttys1``, ``ttys2``, or ``ttys3`` and it is a PIO in the
  native environment, the PIO base of the Service VM legacy vUART 0 would be the same as the PIO base
  corresponding to :option:`hv.DEBUG_OPTIONS.SERIAL_CONSOLE` in the native environment.
- Otherwise, a PIO base would be assigned to the Service VM legacy vUART 0 from the available PIO list.

This logic is implemented in the configuration tools.</xs:documentation>
      </xs:annotation>
    </xs:element>

```

Reply:
Good catch, thanks!